starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,81 @@
+/*Some necessary adjustments*/
+
+h1, body, html, main {
+    margin: 0;
+}
+
+h1 {
+    color: white;
+    font-size: 4em;
+    font-weight: 400;
+}
+
+/*Import the font and set the default size*/
+
+html {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 18px
+}
+
+img {
+    width: 50%;
+    height: auto;
+    text-align: center;
+    margin-left: 25%;
+    margin-right: 25%;
+}
+
+.justify-center {
+    justify-content: center;
+    display: flex;
+    flex-direction: column;
+}
+
+ul, ol {
+    padding-inline-start: 20px;
+}
+
+ul {
+    list-style-type: none;
+    line-height: 1.5em;
+}
+
+ol {
+    list-style-type: none;
+    font-size: 20px;
+}
+
+ol li:before {
+    content: "-";
```

Comment:
Nice workaround! It's really weird that there's no dashed option for list-styles, so this is the best workaround you can do. Good job!